src/main/java/com/nimbusds/jose/util/JSONObjectUtils.java

```diff
@@ -382,6 +384,33 @@ public static URI getURI(final Map<String, Object> o, final String key)
 		}
 	}
 
+	/**
+	 * Gets a JSON objects array member of a JSON object.
+	 *
+	 * @param o   The JSON object. Must not be {@code null}.
+	 * @param key The JSON object member key. Must not be {@code null}.
+	 *
+	 * @return The JSON object member value, may be {@code null}.
+	 *
+	 * @throws ParseException If the value is not of the expected type.
+	 */
+	public static Map<String, Object>[] getJSONObjectArray(final Map<String, Object> o, final String key)
+			throws ParseException {
+
+		List<Object> jsonArray = getJSONArray(o, key);
+
+		if (jsonArray == null) {
+			return null;
+		}
+
+		try {
+			return jsonArray.toArray(new HashMap[0]);
```

Comment:
I know this is in line with the rest of the code in this file, but man it freaks me out ^^